test/helper.rb

```diff
@@ -148,7 +148,7 @@ def skip_until(y, m, d, msg)
   end
 
   def skip_windows!
-    skip_until 2020, 5, 1, "These have never passed" if windows?
+    skip_until 2020, 7, 1, "These have never passed" if windows?
```

Comment:
I had to do this as well! But luckily with the merger I reverted and rebased master